include/envoy/secret/secret.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <stdexcept>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "envoy/common/exception.h"
+#include "envoy/ssl/context.h"
+
+namespace Envoy {
+namespace Secret {
+
+/**
+ * Secret contains certificate chain and private key
```

Comment:
nit: Please go through and end all comments with full stop ('.').

Reply:
Done